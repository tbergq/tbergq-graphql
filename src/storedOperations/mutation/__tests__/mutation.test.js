// @flow

import { generateTestsFromFixtures } from '@kiwicom/test-utils';

import executeTestQuery from '../../../common/services/executeTestQuery';

jest.mock('../../../common/db/models/StoredOperation', () => ({
  addOperations: jest.fn(input => Promise.resolve(input)),
}));

generateTestsFromFixtures(`${__dirname}/__fixtures__`, input => executeTestQuery(input));
