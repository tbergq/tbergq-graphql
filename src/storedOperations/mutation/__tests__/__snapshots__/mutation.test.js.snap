// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches expected output: createStoredOperations.graphql 1`] = `
~~~~~~~~~~ INPUT ~~~~~~~~~~
mutation {
  createdStoredOperations(storedOperations: [{operationId: "123", text:"lol"},{operationId: "321", text:"lol2"}]) {
    createdOperations {
      operationId
      text
    }
  }
}

~~~~~~~~~~ OUTPUT ~~~~~~~~~~
{
  "data": {
    "createdStoredOperations": {
      "createdOperations": [
        {
          "operationId": "123",
          "text": "lol"
        },
        {
          "operationId": "321",
          "text": "lol2"
        }
      ]
    }
  }
}
`;
